docs/administration/index.md

```diff
@@ -0,0 +1,14 @@
+# Administration
+
+**Deploying REANA on your premises?**
```

Comment:
Perhaps just "Deploying REANA?" since we shall add GCP and other options too.

We could also change "Deploying at scale on your premises using Kubernetes and Helm"